README.md

```diff
@@ -469,6 +469,12 @@ that were available on 2017-01-08:
 The USE flags to use for the Node.js package on Gentoo systems. Defaults to
 ['npm', 'snapshot'].
 
+#### `manage_dep_packages`
```

Comment:
Make it clear in the README that this refers to requirements for an apt HTTPS repository, not nodejs itself.